website/_layout/head.html

```diff
@@ -7,6 +7,10 @@
   {{if hascode}} {{insert head_highlight.html }}   {{end}}
   {{insert style.html}}
   {{isdef title}} <title>{{fill title}}</title>  {{end}}
+
+  <!-- LUNR -->
+  <script src="https://unpkg.com/lunr/lunr.js"></script>
```

Comment:
replace this by the block of 4 script lines, to include not only the client but also the index!